src/Components/PeoplePage/PeoplePage.jsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState, useEffect } from 'react';
+import { getPeople } from '../../Api';
+import { PersonRow } from '../PersonRow';
+import './peoplePage.css';
+
+export const PeoplePage = () => {
+    const [people, setPeople] = useState([]);
+
+    const findParents = (array) => {
+        const result = array.map((person) => {
+            person.mother = array.find(({ name }) => name === person.motherName);
```

Comment:
```suggestion
    const findParents = (array) => {
      const result = array.map((person) => {
        person.mother = array.find(({ name }) => name === person.motherName);
```
Too many indentations. Pay attention to formatting